src/Dennis/Link/Checker/Analyzer.php

```diff
@@ -194,6 +194,11 @@ public function getInfo($url) {
    * @throws ResourceFailException
    */
   protected function doInfoRequest($url) {
+    // If url is protocol neutral, force it to use https.
+    if (substr( $url, 0, 2 ) === "//") {
+      $url = ltrim($url, '//');
```

Comment:
Ok, but you could save a line here, shurely?

```
    if (substr( $url, 0, 2 ) === "//") {
      $url = 'https:' . $url;
    }
```

(or `$url = 'http:' . $url;` depending on your thoughts on my previous comment).